cnf-certification-test/lifecycle/suite.go

```diff
@@ -316,6 +322,15 @@ func testDeploymentScaling(env *provider.TestEnvironment, timeout time.Duration)
 	defer env.SetNeedsRefresh()
 	failedDeployments := []string{}
 	for i := range env.Deployments {
+		if scaling.IsManaged(env.Deployments[i].Name, env.Config.ManagedDeployments) {
+			if !scaling.CheckOwnerReference(env.Deployments[i].GetOwnerReferences(), env.Config.CrdFilters) {
```

Comment:
return a list of owner references for the object in the GetOwnerReferences function so that we can include it in the test output/claim